src/main/kotlin/frc/robot/subsystems/conveyor/ConveyorIOReal.kt

```diff
@@ -1,27 +1,54 @@
 package frc.robot.subsystems.conveyor
 
-import com.ctre.phoenix6.configs.Slot0Configs
+import com.ctre.phoenix6.configs.*
 import com.ctre.phoenix6.controls.VelocityVoltage
 import com.ctre.phoenix6.hardware.TalonFX
+import com.ctre.phoenix6.signals.InvertedValue
+import com.ctre.phoenix6.signals.NeutralModeValue
 import edu.wpi.first.units.Angle
 import edu.wpi.first.units.Measure
 import edu.wpi.first.units.Units
 import edu.wpi.first.units.Velocity
 import frc.robot.Ports
+import frc.robot.lib.LoggedTunableNumber
 
 class ConveyorIOReal : ConveyorIO {
     override val inputs = LoggedConveyorInputs()
     private val roller = TalonFX(Ports.Conveyor.MOTOR_ID)
     private val control = VelocityVoltage(0.0).withEnableFOC(true)
 
     init {
-        roller.configurator.apply(ConveyorConstants.MOTOR_CONFIG)
+        val config = TalonFXConfiguration()
+            .withMotorOutput(
+                MotorOutputConfigs()
+                    .withNeutralMode(NeutralModeValue.Coast)
+                    .withInverted(InvertedValue.Clockwise_Positive)
+            ).withSlot0(
+                Slot0Configs()
+                    .withKP(ConveyorConstants.GAINS.kP)
+                    .withKI(ConveyorConstants.GAINS.kI)
+                    .withKD(ConveyorConstants.GAINS.kD)
+                    .withKS(ConveyorConstants.GAINS.kS)
+                    .withKV(ConveyorConstants.GAINS.kV)
+                    .withKA(ConveyorConstants.GAINS.kA)
```

Comment:
Maybe you can get rid of these if they are updated in `periodic`? IDK if `isChanged` would run on the first time
At least you can call `setPID` (which should also be renamed to something like `setGains`) in the subsystem's `init`

Reply:
ischanged will not be called on the first time i tested it.